cmd/keycloakb/keycloak_bridge.go

```diff
@@ -235,6 +235,13 @@ func main() {
 		}
 	}
 
+	// Security - AES encryption mechanism for users PII
+	aesEncryption, err := security.NewAesGcmEncrypterFromBase64(c.GetString("db-aesgcm-key"), c.GetInt("db-aesgcm-tag-size"))
```

Comment:
Please check my PR: https://github.com/cloudtrust/keycloak-bridge/pull/217/files#diff-87b64553cc8dcbe43a0fce63c71a5c71R81
You can declare CfgDbAesGcmKey and CfgDbAesGcmTagSize constants and declare/initialize the configuration in func config(...)

Reply:
Referenced PR is merged...